src/components/activityEventElems/v1/V1ConfigureEventElem.tsx

```diff
@@ -35,6 +38,8 @@ export default function V1ConfigureEventElem({
   return (
     <ActivityEvent
       event={event}
+      withProjectLink={withProjectLink}
+      pv={PV_V1}
```

Comment:
some events are PV-specific and don't have a PV property in the subgraph entity. so we can just hard-code the PV value